WcaOnRails/db/migrate/20170517174106_add_report_nag_sent_at_to_competitions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddReportNagSentAtToCompetitions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :Competitions, :report_nag_sent_at, :datetime
```

Comment:
I think it makes sense to put this on the delegate report instead. Our poor Competitions table has a lot of stuff on it.